pkg/asset/machines/aws/awsmachines.go

```diff
@@ -99,6 +100,11 @@ func GenerateMachines(clusterID string, in *MachineInput) ([]*asset.RuntimeFile,
 		}
 		awsMachine.SetGroupVersionKind(capa.GroupVersion.WithKind("AWSMachine"))
 
+		// // Set Public IP to use BYO IP
+		// if len(in.PublicIpv4Pool) > 0 {
+		// 	awsMachine.Spec.PublicIpv4Pool = ptr.To(in.PublicIpv4Pool)
+		// }
+
```

Comment:
should this be removed?

Reply:
`PublicIpv4Pool` is commented out in a few places.

Reply:
yes, it is old version. Thanks